### ISSUE TITLE
Define format of uploaded maps

### ISSUE THREAD

**Issue by valitsa** [valitsa/ESPmapping#15]
Do we stick to the zipped shapefile and geotiff?
Will we test the rasterised png (is this what we said)? 
Which other formats are we considering?

**Comment by willtemperley** [comment]
I'd like to support NetCDF4 but geoserver doesn't seem to support this at the moment.
http://geoserver.org/display/GEOS/Multidimensional+WCS

Georeferenced PNG / JPG / TIFF should be possible but will probably mean a new way of serving images.

What did people ask for at the conference?

We're somewhat limited by what Geoserver can support too.
http://docs.geoserver.org/stable/en/user/production/data.html

**Comment by valitsa** [comment]
There are 2 reasons for which we discussed that:
a. Improving the visualisation of the uploaded maps
b. Allowing users to upload a 'lighter' version of their maps. 

People didn't really comment on a preferred format at the workshop. 

Raster files and shapefiles are the most commonly used formats, so the zipped shapefile and geotiff option are ok for now as long as we can improve the visualisation (colors, legend). 
We discussed (with you and Bart) the georeferenced png/jpg/tiff for the 'lighter' option. 
Netcdf is less commonly used, so there's no need for that.
